docs/api/handleAction.md

```diff
@@ -29,13 +29,24 @@ import { handleAction } from 'redux-actions';
 
 If a `reducer` function is passed, it is used to handle both normal actions and failed actions. (A failed action is analogous to a rejected promise.) You can use this form if you know a certain type of action will never fail, like the increment example above.
 
+The reducer function gets recieves the following arguments
+
+1. `state`: The current redux state
+
+1. `action`: The redux action
```

Comment:
Numbering here is incorrect, should be `2`. Also I would say

> the Flux Standard Action

instead of 

> the redux action

Reply:
PS markdown suggests
1. A
1. B
1. C


```
1. A
1. B
1. C
```
as it will auto increment numbers for you, and makes adding a step before/after later on not so tedious reordering all of the numbers